src/mmw/js/src/core/models.js

```diff
@@ -53,45 +52,44 @@ var MapModel = Backbone.Model.extend({
         this.set('areaOfInterest', _.clone(this.get('previousAreaOfInterest')));
     },
 
-    setDoubleHeaderSmallFooterSize: function(fit) {
-        this._setSizeOptions({ double: true  }, { small: true }, fit);
-    },
-
-    setDoubleHeaderHalfFooterSize: function(fit) {
-        this._setSizeOptions({ sidebar: true  }, { sidebar: true }, fit);
-    },
-
-    setDoubleHeaderSidebarSize: function(fit) {
-        this._setSizeOptions({ sidebar: true  }, { sidebar: true }, fit);
-    },
-
-    setNoHeaderSidebarSize: function(fit) {
-        var noHeader = true;
-        this._setSizeOptions({ sidebar: true  }, { sidebar: true }, fit, noHeader);
+    setNoHeaderSidebarSize: function(fit, sidebarWidth) {
+        var hasSidebar = true,
+            hasProjectHeader = false;
+        this._setSizeOptions(fit, hasProjectHeader, hasSidebar, sidebarWidth);
     },
 
     setDrawSize: function(fit) {
         this.setNoHeaderSidebarSize(fit);
     },
 
     setAnalyzeSize: function(fit) {
-        this._setSizeOptions({ single: true  }, { large: true }, fit);
+        this.setNoHeaderSidebarSize(fit);
     },
 
     setDataCatalogSize: function(fit) {
-        var wideSidebar = true,
-            noHeader = true;
-        this._setSizeOptions({ sidebar: true  }, { sidebar: true }, fit, noHeader,
-            wideSidebar);
+        this.setNoHeaderSidebarSize(fit, utils.sidebarWide);
+    },
+
+    setModelSize: function(fit) {
+        var hasSidebar = true,
+            hasProjectHeader = true;
+        this._setSizeOptions(fit, hasProjectHeader, hasSidebar);
```

Comment:
It's set to `undefined`, which will set the sidebar to its default width, and not its `-wide` one (there are only two right now). Maybe it'd be better to always pass a `sidebarWidth` mode